define_loudspeakers.c

```diff
@@ -560,7 +556,7 @@ static int lines_intersect(int i,int j,int k,int l,t_ls  lss[MAX_LS_AMOUNT])
   t_float dist_ij,dist_kl,dist_iv3,dist_jv3,dist_inv3,dist_jnv3;
   t_float dist_kv3,dist_lv3,dist_knv3,dist_lnv3;
   // TODO epsilon needs to be updated for 64-bit/double precision
-  t_float epsilon = 1e-9;
+  t_float epsilon = 0.01; // lower precison a bit for floats, was 1e-9;
```

Comment:
i don't know the implications of this, but it seems that the change is rather invasive.

Reply:
Yeah, I'm sure how necessary this really is but I didn't wan to "jinx" a running system.

Reply:
This might be better as an overridable define or event a run time message.

Reply:
Now using FLOAT_EPSILON.